Gemfile

```diff
@@ -13,6 +13,16 @@ gem "ohai", git: "https://github.com/chef/ohai.git", branch: "master"
 gem "chef-config", path: File.expand_path("../chef-config", __FILE__) if File.exist?(File.expand_path("../chef-config", __FILE__))
 gem "cheffish", "~> 14"
 
+# The chef_core gems are sourced from git.
+# gem "chef_core", git: "https://github.com/chef/chef_core", branch: "chef-core-split"
+# gem "chef_core-actions", git: "https://github.com/chef/chef_core-actions", branch: "chef-core-split"
+# gem "chef_core-cliux", git: "https://github.com/chef/chef_core-cliux", branch: "chef-core-split"
+# Temporary for testing:
+gem "train", path: "../train"
```

Comment:
you're going to want train-core here not the full blown train

Reply:
We'll need to provide a train-winrm gem or similar - train-core does not have any transports included.

Reply:
https://github.com/inspec/train/pull/433 will resolve this - once it lands and once after inspec-core 4 release, we can switch to train-core.

Reply:
Based on review discussions, we'll be merging this with train 2.0, but switching over to train-core as soon as it becomes available (eg inspec 4 is available)